### ISSUE TITLE
ef core 3.1 whether support or not comments on the table and columns

### ISSUE THREAD

**Issue by Seanxwy** [dotnet/efcore#21893]
ef core 3.1 whether support or not comments on the table and columns?

**Comment by roji** [comment]
Yes, EF Core 3.1 does support setting comments on [tables](https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationalentitytypebuilderextensions.hascomment?view=efcore-3.1#Microsoft_EntityFrameworkCore_RelationalEntityTypeBuilderExtensions_HasComment__1_Microsoft_EntityFrameworkCore_Metadata_Builders_EntityTypeBuilder___0__System_String_) and [columns](https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationalpropertybuilderextensions.hascomment?view=efcore-3.1#Microsoft_EntityFrameworkCore_RelationalPropertyBuilderExtensions_HasComment__1_Microsoft_EntityFrameworkCore_Metadata_Builders_PropertyBuilder___0__System_String_):

Here's a quick code sample:

```c#
protected override void OnModelCreating(ModelBuilder modelBuilder)
{
    modelBuilder.Entity<Blog>(b =>
    {
        b.HasComment("Table comment");
        b.Property(b => b.Name).HasComment("Column comment");
    });
}
```
